cmd/notary/main.go

```diff
@@ -25,7 +25,7 @@ const tufDir string = configPath + "tuf/"
 
 var caStore trustmanager.X509Store
 var certificateStore trustmanager.X509Store
-var privKeyStore trustmanager.FileStore
+var privKeyStore *trustmanager.KeyFileStore
```

Comment:
This should probably be an interface type with KeyFileStore implementing the interface. It will make it more flexible if we want to add other types of key store in the future.